mark-jacob/starter-code/styles/modules.css

```diff
@@ -0,0 +1,79 @@
+.site-header {
+  background: #FFEAAA;
+}
+
+/* REVIEW: Float the main header to the left, to make room for the hamburger menu. */
+.heading-group-h1 {
+  float: left;
+  padding: 0 0 .2em 1em;
+}
+
+/* DONE: Add styles for the icon-menu class. It should be bigger, and off to the right. */
+.icon-menu:before {
+  content: "\e902";
+  font-size: 5vw;
+}
+.main-nav {
+  padding: 0.1em;
+  text-align: right;
+  padding-right: 1vw;
+}
+
+.main-nav ul {
+  display: none;
+  padding: 0;
+  list-style: none;
+  clear: both;
+}
+
+.main-nav:hover ul {
+  display: block;
+}
+
+.main-nav li {
+  margin-right: .4em;
+  font-size: 1.5em;
+  padding: .5em .2em;
+  border-bottom: 1px solid #ccc;
+}
+
+#filters {
+  padding: 0;
+  list-style: none;
+}
+
+.template {
+  display: none;
+}
+
+.byline {
+  color: #777;
+}
+
+.byline a {
+  font-weight: bold;
+  text-decoration: none;
+}
+
+/* DONE: Now add styles for how things should look on a larger screen:
+         - Hide the hamburger.
+         - Main nav should be all on one row, with padding spaced better for a larger screen.
+         - Just, ya know, try to make it look good! */
+@media all and (min-width:800px) {
+  .icon-menu {
+  visibility: hidden;
+  }
+  .main-nav ul {
+    display: inline-flex;
+  }
+  .main-nav li {
+    margin-right: .4em;
+    font-size: 1.8em;
+    padding: .5em .2em;
+    border-bottom: 1px solid #ccc;
+  }
+  .main-nav{
```

Comment:
on hover your nav is changing from displaying the li all on one line to displaying them vertically, you'll want to add conditions in your media queries to override the hover.